wapi/curves.py

```diff
@@ -214,7 +219,10 @@ def get_instance(self, tag, issue_date, with_data=True, data_from=None, data_to=
         result = self._load_data(url, 'Failed to load tagged instance')
         if result is None:
             return result
-        return util.TS(input_dict=result, tag=tag, issue_date=issue_date, curve_type=util.TAGGED_INSTANCES)
+        res = [util.TS(input_dict=r, issue_date=issue_date, curve_type=util.TAGGED_INSTANCES) for r in result]
+        if unwrap and len(res) == 1:
+            res = res[0]
```

Comment:
Isn't it better to always return an array, and not sometimes return a TS instead?

Reply:
It would be more regular, yes.  The main reason to unwrap when a single tag is requested, is that it becomes backwards compatible with previous behaviour.

Reply:
Is it important to keep backwards compatibility this early on?

Reply:
It's also a bit about the path of least surprise, a get of a fully specified single time series returns the series.  Let's start this way, and then keep an eye on it to see if it's a mistake or not.